### PR TITLE
Verify table existance on dashboard template

### LINKED ISSUE #44: Dashboard crashes after removing Posts
I've removed Posts table from database manager and dashboard crashed. 

> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'voyager.posts' doesn't exist (SQL: select count(*) as aggregate from `posts`)

There is no views when i publish vendor files. and no option to customize dashboard view.
Would be good if you make possible to publish views

### PR DESCRIPTION
**Pull request by fletch3555**
Fixes #44

### CODE REVIEW

**Comment by fletch3555** [comment]
This hides the panels on the homepage if the Post and Page tables don't exist.  There may be a cleaner way to do this, but I wasn't able to find it.

**Comment by marktopper** [comment]
Nice approach!
Will just test this later before merging.
Rock on 🤘

File: resources/views/index.blade.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 @section('content')
     <div class="page-content">
         <div class="widgets">
+            <?php if (Illuminate\Support\Facades\Schema::hasTable(with(new TCG\Voyager\Models\User())->getTable())) { ?>
```
[the remark this answers is not in the document]
Comment (fletch3555) [reply resources/views/index.blade.php:10]: I believe you're right.  I can pull that out.

[the remark this answers is not in the document]
Comment (marktopper) [reply resources/views/index.blade.php:10]: It's okay. We might change it later. Merge coming right up.

**Pull request merged**